diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -444,17 +447,17 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
      *
      * @return true - if there is error, and false if there is no error
      */
-    private fun ASTNode.isCallsChain(isWithoutParentheses: Boolean = true) = getCallChain(isWithoutParentheses)?.isNotValidCalls(this) ?: false
+    private fun ASTNode.isCallsChain(dropLeadingBrackets: Boolean = true) = getCallChain(dropLeadingBrackets)?.isNotValidCalls(this) ?: false
 
-    private fun ASTNode.getCallChain(isWithoutParentheses: Boolean = true): List<ASTNode>? {
+    private fun ASTNode.getCallChain(dropLeadingBrackets: Boolean = true): List<ASTNode>? {
```

Comment:
We are not dropping brackets, we are dropping parts of dot-qualified expression which are not function calls.